### PR TITLE
Update tsconfig and web help deps

### PR DESCRIPTION
**Pull request by t1m0thyj**
⚠️ Don't merge until https://github.com/zowe-actions/octorelease/pull/147 is published

**What It Does**

* Enable declaration maps in tsconfig so that "Go to Definition" goes to `.ts` files instead of `.d.ts` files
* Update web help dependencies for technical currency (including major update from Bootstrap 4 to 5)
* Fix active node not updating in web help tree when browsing as a local file

**How to Test**

To test the tsconfig fix:
1. Check out this branch and don't rebuild yet.
1. Open a source file that imports classes from other SDKs (such as `packages/zosuss/src/SshBaseHandler.ts`).
1. Ctrl+Click on one of the imported class names (e.g. `ImperativeError`) and notice a `.d.ts` file is opened.
1. Rebuild , then Ctrl+Click the class name again and notice a `.ts` file is opened.

To test changes to web help, click on links in the doc and verify that active (highlighted) item in tree updates:
test-web-help-cropped

**Review Checklist**
I certify that I have:
- [x] tested my changes
- [ ] added/updated automated tests
- [x] updated the changelog
- [x] followed the [contribution guidelines](https://github.com/zowe/zowe-cli/blob/master/CONTRIBUTING.md)

**Additional Comments**

### CODE REVIEW

**Review by zFernand0 — APPROVED** [review]
LGTM! 😋

**Review by traeok — APPROVED** [review]
LGTM, thanks Timothy for the updates!

File: packages/imperative/CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,14 @@
...
 
+## Recent Changes
+
+- BugFix: Fixed issue in local web help with highlighted sidebar item getting out of sync. [#2215](https://github.com/zowe/zowe-cli/pull/2215)
+- BugFix: Updated web help dependencies for technical currency. [#2215](https://github.com/zowe/zowe-cli/pull/2215)
+
 ## `5.26.1`
 
-- Bugfix: Export new Proxy class from Zowe imperative package. [#2205](https://github.com/zowe/zowe-cli/pull/2205)
+- BugFix: Export new Proxy class from Zowe imperative package. [#2205](https://github.com/zowe/zowe-cli/pull/2205)
```
Comment (anaxceron) [inline packages/imperative/CHANGELOG.md:12]: How is this a bug fix? Why did this need to be addressed?

Comment (t1m0thyj) [reply packages/imperative/CHANGELOG.md:12]: Good catch, this is a release note from a previous PR but I'll update it to match what we have on the vNext branch:
> - BugFix: Fixed missing export for `Proxy` class in Imperative package.

Pushed the update on another PR: https://github.com/zowe/zowe-cli/pull/2218

**Review by gejohnston — APPROVED** [review]

**Review by anaxceron — APPROVED** [review]

**Review by zFernand0 — APPROVED** [review]
Clever way to allow the rewriteShrinkwrap.js file to be executed from a different subpackage on the monorepo 😋

**Review by awharn — APPROVED** [review]

**Pull request merged**